src/PoolManager.sol

```diff
@@ -440,6 +442,10 @@ contract PoolManager is Auth {
         return tranche.token;
     }
 
+    function getTrancheTokenRestriction(uint64 poolId, bytes16 trancheId) public view returns (uint8 restrictionSet) {
```

Comment:
I wouldn't add this method just for tests. It's also a bit confusing because the name implies it always gets the restriction, but it actually only works for undeployed tranches